src/App.js

```diff
@@ -1,22 +1,14 @@
-import logo from './logo.svg';
 import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
 
```

Comment:
- [ ] Your application is not starting due to **improper Eslint configuration**. Kindly ensure to improve upon the same. You can optionally install  clean linter from [Microverse Eslint](https://github.com/microverseinc/linters-config/tree/master/react-redux)

**Eslint configuration errors:**
![image](https://user-images.githubusercontent.com/12745474/142950494-f323bcb5-4e32-438c-94e2-57ea245b251b.png)

![image](https://user-images.githubusercontent.com/12745474/142950571-f4c2ea77-4ccc-4249-9d37-6baf7fa378d7.png)

Reply:
- [ ] the issue still persists, please search on the internet how to fix it.

![Capture web_23-11-2021_1469_localhost](https://user-images.githubusercontent.com/61562626/143029514-a8779424-55b6-449c-b45a-36997d94ef9c.jpeg)